collectors/python.d.plugin/ipfs/ipfs.chart.py

```diff
@@ -37,7 +30,7 @@
         ]
     },
     'repo_size': {
-        'options': [None, 'IPFS Repo Size', 'GB', 'Size', 'ipfs.repo_size', 'area'],
+        'options': [None, 'IPFS Repo Size', 'GiB', 'Size', 'ipfs.repo_size', 'area'],
         'lines': [
             ['avail', None, 'absolute', 1, 1e9],
             ['size', None, 'absolute', 1, 1e9],
```

Comment:
All of these lines should have `1 << 30` as the final value, not `1e9`, otherwise we're labeling them with the wrong units.